README.md

```diff
@@ -15,16 +15,20 @@ Then install the framework into the `lib/` directory using:
 
 ```
 mkdir lib/
-pip install -t lib/ https://github.com/canonical/operator
+pip3 install -t lib/ https://github.com/canonical/operator/archive/master.zip
```

Comment:
I can confirm that this works for me (on bionic) with --system added.

I'm curious why you chose .../archive/master.zip instead of `pip3 install --system -t lib/ git+https://github.com/canonical/operator`

Both seem to work for me, so its more a question of why one would be preferred over the other.

Reply:
I actually had the `git+` style a moment before you added this comment, but I saw https://github.com/johnsca/operator/pull/1 where @timClicks made the claim that it would be faster.  I don't have any evidence for or against, but it does seem like it might avoid using VCS so could potentially be faster or more portable.  I have no preference personally on which we use.